apps/ichat-gw/MediaRelay.cxx

```diff
@@ -330,10 +332,10 @@ MediaRelay::checkKeepalives(MediaRelayPort* relayPort)
       //}
 
       // Add message to buffer
-      std::auto_ptr<char> buffer(new char[sizeof(RtpHeader)]); 
+      std::unique_ptr<char> buffer(new char[sizeof(RtpHeader)]);
```

Comment:
do you have to use `std::unique_ptr<char[]>` here to invoke `delete []`?

Reply:
Yes, I believe so.